### PR TITLE
Readme 09 15

### PR DESCRIPTION
**Pull request by perryr16**
## What's this PR do?
* Update Readme
* Update splash page to show available routes
## Where should the reviewer start? 
* ReadMe 
* app.py
## How should this be manually tested?
* localhost:300/
## Any background context you want to provide?
* n/a
## What are the relevant tickets?
* n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,5 +15,4 @@ deploy:
   api_key:
     secure: qofM3BtlKhNbnirVWpHS15WPXDWH88MNEbo8AyVM+Y08AvzDUdbgbHQxtPn07Ke1K0X2ZexqNLV6TDbatvIW3nXxPm23DtSySb9DVpegpW/TD+C0ryO4emta3eaRmOIbnt+Aq8XSKnOBr6BOchW9EKJnFmCexJqiq935SarELtmU+J3LncEwZQuTQyVs8UpwBFtOryBkugFwRVhMjpHYqChsu+bgsDToMbjCZp8YD5e9GHo6j2Jqf3DJ+/z9UBKvtuF1brUlv8tOxIDnmdCDcaOLkewifSafn5GjtYGjJNVlqOL5mQsLEYMmWrRT7kxiVWbW08hsW1XT3ZGmhHg3nDOsYFqOGh0ezV7wD5D4/4FfWr+R26E8yXkTZXUJNsgihs6IjvDVreeRsgTe8SpfcEHViE75qddernKIPQywXPLqkUzo4aOug9Eb+2rYgKRQMCj91WDqNTiuWAAuBxrhgpyY5vgI31bZv21dhqYALMH9KrS4ZztN8qfM18wCHhXju5qOqT1mocgZoVyK5piBKcwijPQHhIn0vpjXY70GsvWO8T3gXBSLv1ucpIfnHjLW46LU0oqimkWQnpMYtHtfDAituLiCQsC8VRzZHyrzRAe7bE5DNeByPRjmALJc2HmLT2rp8TR6B9KG3mo9dpx5Fbtm21sohFDaZp2nsWrefuM=
   app: etho-boost
-  run: python3 manage.py db upgrade
```
Comment (perryr16) [inline .travis.yml:18]: Lets not mess with heroku right now, although auto-deployment is not active

File: README.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+- [About](#About)
 - [Setup](#setup)
 - [Database Setup](#database-setup)
 - [Seeding The Database](#seeding-the-database)
 - [Updating The Database](#database-updates)
 - [API Endpoints](#api-endpoints)
 
+## About <a name="about"></a>
+Etho-Boost is a crowdfunding platform built on the Ethereum Blockchain. View the production [frontend](https://etho-boost-crowdfund.herokuapp.com/) or [backend](https://etho-boost.herokuapp.com/)
+
```
Comment (perryr16) [inline README.md:11]: Needs some work

File: src/app.py [hunk 3]
```diff
@@ -31,6 +31,20 @@ def create_app(env_name):
...
+              'All Requests': 'GET: /api/v1/requests/', 
+              'Create Request': 'POST: /api/v1/requests/', 
+              'Update Request': 'PUT: /api/v1/requests/<request_id>',
+              'Request by ID': 'GET: /api/v1/requests/<request_id>',
+              'Create Contributor': 'POST: /api/v1/contributor/', 
+              'Update Contributor': 'PUT: /api/v1/contributor/<contributor_id>', 
+              'Contributor by contributor id': 'GET: /api/v1/contributor/<contributor_id>', 
+              'Price Conversion': 'GET: /api/v1/price_converter_api/?wei=<wei_amount>`',
+              }
+    return message
```
Comment (perryr16) [inline src/app.py:48]: Message displayed on splash page

Comment (takeller) [reply src/app.py:48]: Great Idea!

**Review by takeller — APPROVED** [review]

**Review by AntonioJacksonII — APPROVED** [review]
Nice updates.

**Pull request merged**